Tests/ClientTests/RemoteTabsPanelTests.swift

```diff
@@ -8,6 +8,8 @@ import Common
 @testable import Client
 
 class RemoteTabsPanelTests: XCTestCase {
+    let tabsEnabledPrefKey = "sync.engine.tabs.enabled"
```

Comment:
Why do you have this separate, if it's not a separate string? We should just import shared and have access to the PrefsKey that way.